Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -176,15 +176,16 @@ public class PythonCompiler : SinglePassCompiler {
         : "";
       var methodWriter = wr.NewBlockPy(header: $"class {IdProtect(name)}{baseClasses}:");
 
-      var needsConstructor = cls is TopLevelDeclWithMembers decl
-                             && decl.Members.Any(m => !m.IsGhost && ((m is Field && !m.IsStatic) || m is Constructor));
-      ConcreteSyntaxTree constructorWriter = null;
-      if (needsConstructor) {
-        var block = methodWriter.NewBlockPy(header: "def  __init__(self):", close: BlockStyle.Newline);
-        constructorWriter = block.Fork();
-        block.WriteLine("pass");
+      var args = typeParameters.Where(NeedsTypeDescriptor).Comma(tp => tp.CompileName);
```

Comment:
Let's add a variable for `typeParameters.Where(NeedsTypeDescriptor)` to make it obvious that they can't get out of sync